etc/di.xml

```diff
@@ -90,6 +90,12 @@
             <argument name="logger" xsi:type="object">AmwalLogger</argument>
         </arguments>
     </type>
+    <type name="Amwal\Payments\Model\Config">
```

Comment:
I don't think it's necessary to add these arguments to di as they are fixed in the constructor of the class already